src/Operations/evolve.jl

```diff
@@ -11,6 +11,11 @@ struct ZipUpAlgorithm <: EvolveAlgorithm end
 
 function evolve! end
 
+function evolve(tn, op; kwargs...)
+    tn = copy(tn)
+    evolve!(tn, op; kwargs...)
+end
+
```

Comment:
can be done in one line


```suggestion
evolve(tn, op; kwargs...) = evolve!(copy(tn), op; kwargs...)
```